pkg/client/group_mgmt_client_test.go

```diff
@@ -108,3 +110,40 @@ func TestListGetOrPost(t *testing.T) {
 
 	// List Volumes with SearchFilter
 }
+func TestVolumeAvgStatLast5mins(t *testing.T) {
+        respHolder := &nimbleos.Volume{}
+        body := []byte(`
+         {
+                "data": {
+                   "id": "0644aeb604d6b5e7c6000000000000000000000002",
+                   "name": "hiqa-win25-ESX-Host-sjc-array2094-datastore-for-VMs",
+                   "pool_name": "default",
+                   "size": 7340032,
+                   "usage_valid": true,
+                   "vol_usage_mapped_bytes": 301340909568,
+                   "reserve": 0,
+                   "snap_usage_uncompressed_bytes": 0,
+                   "avg_stats_last_5mins": {
+                          "combined_iops": 192,
+                          "combined_latency": 69,
+                          "combined_throughput": 2974470,
+                          "read_iops": 10,
+                          "read_latency": 155,
+                          "read_throughput": 1529765,
+                          "write_iops": 180,
+                          "write_latency": 70,
+                          "write_throughput": 1444702
+                   }
+                }
+         }`)
+
+        dataIntf, totalRows, err := unwrapData(body, respHolder)
+
+        assert.Nil(t, err)
+        assert.Nil(t, totalRows)
+        assert.NotNil(t, dataIntf)
+		volume := dataIntf.(*nimbleos.Volume)
```

Comment:
gofmt would clean up the inconsistent space/tab characters.

Reply:
thanks . done